console-api/proto/tasks.proto

```diff
@@ -49,6 +50,9 @@ message TaskDetails {
 
     // HdrHistogram.rs `Histogram` serialized to binary in the V2 format
     optional bytes poll_times_histogram = 3;
+
+    // What the task is currently busy doing.
+    repeated consequences.Causality causality = 4;
```

Comment:
i don't love calling this field `causality` --- that suggests to me that a task's `causality` is that which _caused_ the task to be spawned. i might call this something like `spans` or `trace` or something, since it's specifically a list of the span trees that the task is in, i believe?